src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -163,6 +191,206 @@ export default ({
     }
   }, [camera, engineAPI]);
 
+  const geodsic = useMemo(():
+    | undefined
+    | { geodesicVertical: EllipsoidGeodesic; geodesicHorizontal: EllipsoidGeodesic } => {
+    const viewer = cesium.current?.cesiumElement;
+    if (
+      !viewer ||
+      !viewer.scene ||
+      viewer.isDestroyed() ||
+      !property?.cameraLimiter ||
+      !property?.cameraLimiter.cameraLimitterTargetArea
+    )
+      return undefined;
+    const ellipsoid = viewer.scene.globe.ellipsoid;
+
+    const centerPoint = Cartesian3.fromDegrees(
+      property.cameraLimiter.cameraLimitterTargetArea.lng,
+      property.cameraLimiter.cameraLimitterTargetArea.lat,
+      0,
+    );
+
+    const CartographicCenterPoint = Cartographic.fromCartesian(centerPoint);
```

Comment:
```suggestion
    const cartographicCenterPoint = Cartographic.fromCartesian(centerPoint);
```

First char uppercase is used for classes in JS